critiquebrainz/frontend/views/review.py

```diff
@@ -212,10 +212,6 @@ def create():
             entity_type = supported_type
             break
 
-    if not (entity_id or entity_type):
```

Comment:
I think that this check is useful, because in CB you can't review all types of entities in MB.
I'd like to keep this check, but perhaps we could improve the way that we format the URL, e.g. instead of /write?artist=id, what if we had `/write/artist/id`? This would make it easier to get the variable for the entity type, and would fix a bug that exists where you can add 2 different entity types as parameters!

We could do something like:
```py
@route('/write/<entity_type>/<entity_id>/)
def create(entity_type, entity_id):
    if entity_type not in ENTITY_TYPES:
        raise BadRequest
    if not entity_id:
        redirect
    ... other checks
```

In this case I'd also like to see tests for these checks, and we should also have a redirect endpoint that converts old-style urls into new-style